service/pom.xml

```diff
@@ -49,6 +49,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
```

Comment:
We already depend on it through gcm-server, so why not ? 
And if not, what should be used instead ? I tried jackson, but it wasn't... simple :smile: